src/Surfnet/StepupSelfService/SamlStepupProviderBundle/DependencyInjection/Compiler/ViewConfigCollectionPass.php

```diff
@@ -43,6 +43,7 @@ public function process(ContainerBuilder $container)
         $taggedServices = array_keys($taggedServices);
 
         foreach ($taggedServices as $id) {
+            $gsspIdMatches = null;
```

Comment:
I know this is possible but should be an empty array. Feel free to remove or leave as-is.